options/path.c

```diff
@@ -76,10 +76,14 @@ static const char *mp_get_platform_path(void *talloc_ctx,
     assert(talloc_ctx);
 
     if (global->configdir) {
+        // Return NULL for all platform paths if --no-config is passed
+        if (global->configdir[0] == '\0')
```

Comment:
trivial suggestion if you want to bother
```suggestion
        if (!global->configdir[0])
```

Reply:
Sure